internal/services/datafactory/data_factory_linked_service_sftp_resource.go

```diff
@@ -103,31 +121,42 @@ func resourceDataFactoryLinkedServiceSFTP() *pluginsdk.Resource {
 				},
 			},
 
-			"skip_host_key_validation": {
-				Type:     pluginsdk.TypeBool,
-				Optional: true,
+			"passphrase": {
+				Type:          pluginsdk.TypeString,
+				Optional:      true,
+				Description:   "Specify the pass phrase or password to decrypt the private key if the key file or the key content is protected by a pass phrase",
```

Comment:
Maybe we could use one word here in the description to keep it more concise?
```suggestion
				Description:   "Specify the password to decrypt the private key if the key file or the key content is password protected",
```

Reply:
Thanks! Yeah I just copy-pasta'd from Microsoft's docs, I'm all for making it more concise.